src/ui/Help.tsx

```diff
@@ -85,8 +86,19 @@ export const Help = ({
                 e.target.value || initialOptions.drawColour,
               )
             }
-            placeholder="CSS Colour"
-          />
+          >
+            {Object.entries(customColorOptions).map(([key, value]) => {
```

Comment:
1. Lodash `_.map` rather than `Object.entries`, since we've got it :)
2. Stylistic thing, but I'd always rather see `(...) => (value)` over `(...) => { return (value) }`